src/components/settings-menu/BackupSection/AlreadyBackedUpView.js

```diff
@@ -266,7 +268,7 @@ export default function AlreadyBackedUpView() {
           <SheetActionButton
             androidWidth={225}
             color={colors.white}
-            label="🗝 View recovery key"
+            label="🗝 View secret phrase"
```

Comment:
Here we should check the wallet type and show the correct phrase (secret phrase or private key)

Reply:
good 2 go: http://cloud.skylarbarrera.com/Screen-Recording-2021-05-25-23-30-58.mp4